specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,539 @@
+    pub updated_at: Option<String>, // ISO 8601 datetime when token was last updated
+    pub extra: Option<String>, // anything extra the NFT wants to store on-chain. Can be stringified JSON.
+    pub reference: Option<String>, // URL to an off-chain JSON file with more info.
+    pub reference_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of JSON from reference field. Required if `reference` is included.
+}
+
+/// Offers details on the  metadata.
+pub trait MultiTokenMetadataProvider {
+    fn mt_metadata(&self, token_id: TokenId) -> MultiTokenMetadata;
+}
+```
+# Drawbacks
+[drawbacks]: #drawbacks
+Doing this adds another spec and codebase to the standards. It could be seen that we could leave this to developers to implement custom solutions and have them create a token that just uses FT and NFT together. There is some additional complexity
+in ux, when considering batch size request and gas limitations, that might trip some developers up. 
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+The rationale for this design, is that we want to support developers that need batch requests. In many ecosystems, a single NFT or 
+series of NFT are not sufficient for representing and managing potentially 100s of tokens. Managing cross contract calls and the token contracts themselves become a huge burden on the developer. This will reduce the complexity, and allow developers to easily bridge over to other contracts like ERC-1155 on other chains that allow for a this style of representation.
+
+In the design phase it was considered to simply rely on the underlying implementations of fungible tokens and non-fungible tokens
+to be the scope of interaction with the chain. Doing this we would have tied the implementations to the bounds of FT and NFT. By loosing this up a bit, we are able to be a bit more flexible in what's possible.
+
+Not doing this means we really won't have a great way of supporting use cases where developers need to represent and manage
+large varying quantities and types of tokens. They will all have to implement it on their own, and that would make for a fragmented
+and weak ecosystem. Where every developer would not be able to reliably trade these assets.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+The unresolved questions, are really what type of metadata , is required for this spec?
+- We decided that having metadata on chain is good, and that at later time we can have a spec for extra data if needed
+
+Can we represent events in this spec, which would improve the ecosystem quite a bit? If we represent events what should those events be?
+- We decided events aren't here yet in the ecosystem, in the way we' like them to be so they are't apart of this standard
+
+Should we have a spec for TokenType?
+- We decided TokenType shouldn't be exposed for the public interface consumption. We will leave this description in the hands of the implementers
```

Comment:
Is `TokenType` how I would know if a particular token_id is FungibleToken or NonFungibleToken?  If so, why would we not want to expose that piece of data?  As a developer intending to interface with the contract, won't I want to know if a particular token is a fungible or non fungible token when I query it?  It will likely influence where a particular token is displayed in the UI of a wallet, for example (is this a Collectible (NFT) or is this a Fungible Token?)

Reply:
just going to makr this here that we can bring this up at next meeting.